build.sbt

```diff
@@ -165,7 +208,17 @@ lazy val mimaSettings = {
     mimaBinaryIssueFilters ++= {
       import com.typesafe.tools.mima.core._
       import com.typesafe.tools.mima.core.ProblemFilters._
-      Seq()
+      Seq(
+        ProblemFilters.exclude[DirectMissingMethodProblem](
+          "io.chrisdavenport.semigroups.First.firstSemigroup"
+        ),
+        ProblemFilters.exclude[DirectMissingMethodProblem](
+          "io.chrisdavenport.semigroups.Intersect.IntersectEq"
+        ),
+        ProblemFilters.exclude[DirectMissingMethodProblem](
+          "io.chrisdavenport.semigroups.Last.lastSemigroup"
+        )
```

Comment:
I didn't notice this in Monoids and don't quite understand why it was necessary here.  It looks like the companions still extend the instances?

Reply:
I guess that's because some breaking changes weren't released (like https://github.com/typelevel/semigroups/pull/59, https://github.com/typelevel/semigroups/commit/c205bf6fdb90d92871dcc21143877f099acddb54#diff-2ea58c8be0fa860b5a4bd45b6ea87cd7d9b5eb148254b5d478851300ff6849e1R19). Mima compares with the `0.2.0` version which doesn't contain that changes and therefore checks fail.

Reply:
If this gets released post-Typelevel migration, it also doesn't matter at all.

Reply:
Yep, that's right.